timml/besselaesnew/besselaesnew.py

```diff
@@ -0,0 +1,706 @@
+import numba
+import numpy as np
+
+
+def initialize():
+    pass
+
+
+@numba.njit
+def make_rbnimon():
```

Comment:
This function appears to be run once on import, is it really necessary to compile it?  Also, this function name appears to have a typo in its name if I compare it with the `RBINOM` constant below.